doc/classes/AnimationNodeBlend3.xml

```diff
@@ -4,11 +4,12 @@
 		Blends two of three animations linearly inside of an [AnimationNodeBlendTree].
 	</brief_description>
 	<description>
-		A resource to add to an [AnimationNodeBlendTree]. Blends two animations together linearly out of three based on a value in the [code][-1.0, 1.0][/code] range.
+		A resource to add to an [AnimationNodeBlendTree]. Blends two animations together linearly out of three based on an amounmt value.
 		This node has three inputs:
-		- The base animation
-		- A -blend animation to blend with when the blend amount is in the [code][-1.0, 0.0][/code] range.
-		- A +blend animation to blend with when the blend amount is in the [code][0.0, 1.0][/code] range
+		- The base animation to blend with
+		- A -blend animation to blend with when the blend amount is negative value
+		- A +blend animation to blend with when the blend amount is positive value
+		In general, the blend value should be in the [code][-1.0, 1.0][/code] range, but you can use outside of that range for blending amplified or inverted animations. However, it may look better to use [AnimationNodeAdd3] for that purpose.
```

Comment:
```suggestion
		- The base animation to blend with
		- A "-blend" animation to blend with when the blend amount is negative
		- A "+blend" animation to blend with when the blend amount is positive
		In general, the blend value should be in the [code][0.0, 1.0][/code] range. Values outside of this range can blend amplified or inverted animations, however,  [AnimationNodeAdd3] works better for this purpose.
```

Reply:
For Blend3, the [-1.0, 1.0] range is not wrong. Corrections have been applied to the rest of the grammar.